workers/data_refinery_workers/processors/test_salmon.py

```diff
@@ -154,16 +154,18 @@ def test_success(self):
         # Another function being tested
         job_context = salmon._download_index(job_context)
 
-        self.assertFalse("success" in job_context)
+        self.assertTrue(job_context["success"])
+        print(job_context)
         self.assertTrue("index_directory" in job_context)
         self.assertTrue(os.path.isdir(job_context["index_directory"]))
         self.assertEqual(9, len(os.listdir(job_context["index_directory"])))
 
         # Another function being tested
         job_context = salmon._run_salmon(job_context)
 
-        self.assertFalse("success" in job_context)
-        self.assertEqual(6, len(os.listdir(output_directory_path)))
+        self.assertTrue(job_context["success"])
+        # What are these files supposed to be? Why magic '6'?
+        # self.assertEqual(6, len(os.listdir(output_directory_path)))
```

Comment:
Mmm, good point. Those files are the output of salmon. This test is mostly just to make sure that salmon actually output a directory, it could be `self.assertGreater(len(os.listdir(output_directory_path), 1)`.